src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -179,14 +200,41 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
     this.setState(stateUpdate);
   }
 
+  private clearVariables(variablesToClear: Array<SceneVariable<SceneVariableState>>) {
+    return () => {
```

Comment:
Minor, but I'm not a huge fan of a class method returning a callback, and it's more like a variables-clearing factory.

Reply:
Fixed!